src/mds/Locker.cc

```diff
@@ -3193,7 +3193,7 @@ void Locker::handle_client_caps(const cref_t<MClientCaps> &m)
      *   - mds trim its cache
      *   - mds receives cap messages from client
      */
-    dout(7) << "handle_client_caps on unknown ino " << m->get_ino() << ", dropping" << dendl;
+    dout(0) << "handle_client_caps on unknown ino " << m->get_ino() << ", dropping" << dendl;
```

Comment:
If this is indeed the problem, I foresee a lot of huge MDS journals when an older kernel is doing async creates. Can we just make this a one-time `dout(0)`?

(Would be nice with some macro magic to make a one-time `dout(0)` message and then fallback to normal level.)

Reply:
Yeah, we have stuff like WARN_ON_ONCE In the kernel that does a similar sort of thing, and it's nice for keeping logs manageable. Let me see what I can come up with.

Reply:
Ok, I added ldout_once and dout_once macros, and converted the code over to use it for these messages.